python/GangaLHCb/test/Unit/DiracAPI/TestDiracCommands.py

```diff
@@ -94,8 +95,11 @@ def dirac_job(load_config):
     """
     api_script = dedent(api_script)
 
+    cred_req = DiracProxy(group='lhcb_user')
```

Comment:
Erm, you may have to copy https://github.com/ganga-devs/ganga/pull/917/files#diff-b063f6a99352c1604f128062072de4f9R161 wrt  `encodeDefaultProxyFileName=False` option as the default config is _not_ consistent with the lhcb expected use which can make running this manually a pain. I don't know if this comes to bite us here as well or not, you may be saved by the fact you're being explicit in passing the cred along to the Dirac layer.